app/main.py

```diff
@@ -1,13 +1,13 @@
-# from models import Actor
```

Comment:
Do not include DB files to the pull request as mentioned in the [checklist](https://github.com/Marinel444/py-actor-manager/blob/develop/checklist.md#dont-push-db-files).